itests/fixtures/boost_block_reward_fixtures.go

```diff
@@ -0,0 +1,207 @@
+		config.WithFeatureSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_p21eqp23.json",
+		),
+		config.WithRewardSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_p21eqp23.json"),
+	}
+}
+
+func (suite *RewardBoostDaoXtnEqualPeriodsPreactivatedAllSuite) SetupSuite() {
+	suite.BaseSetup(suite.BlockchainOpts()...)
+}
+
+// RewardBoostDaoXtnP21LessP23PreactivatedAllSuite use preactivated features 14, 19, 20, 21, 22, 23,
+// 2 miners, dao, xtn, initR = 900000000, increment = 100000000, desiredR = 700000000,
+// min_xtn_buy_back_period < block_reward_boost_period.
+type RewardBoostDaoXtnP21LessP23PreactivatedAllSuite struct {
+	BaseSuite
+}
+
+func (suite *RewardBoostDaoXtnP21LessP23PreactivatedAllSuite) BlockchainOpts() []config.BlockchainOption {
+	return []config.BlockchainOption{
+		config.WithFeatureSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_p21lsp23.json",
+		),
+		config.WithRewardSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_p21lsp23.json"),
+	}
+}
+
+func (suite *RewardBoostDaoXtnP21LessP23PreactivatedAllSuite) SetupSuite() {
+	suite.BaseSetup(suite.BlockchainOpts()...)
+}
+
+// RewardBoostDaoXtnP21GreaterP23PreactivatedAllSuite uses preactivated features 14, 19, 20, 21, 22, 23,
+// 2 miners, dao, xtn, initR = 900000000, increment = 100000000, desiredR = 700000000,
+// min_xtn_buy_back_period > block_reward_boost_period.
+type RewardBoostDaoXtnP21GreaterP23PreactivatedAllSuite struct {
+	BaseSuite
+}
+
+func (suite *RewardBoostDaoXtnP21GreaterP23PreactivatedAllSuite) BlockchainOpts() []config.BlockchainOption {
+	return []config.BlockchainOption{
+		config.WithFeatureSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_p21grp23.json",
+		),
+		config.WithRewardSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_p21grp23.json"),
+	}
+}
+
+func (suite *RewardBoostDaoXtnP21GreaterP23PreactivatedAllSuite) SetupSuite() {
+	suite.BaseSetup(suite.BlockchainOpts()...)
+}
+
+// RewardBoostMinerDaoPreactivatedAllSuite tests the boost block reward distribution with 1 miner and DAO settings.
+type RewardBoostMinerDaoPreactivatedAllSuite struct {
+	BaseSuite
+}
+
+func (suite *RewardBoostMinerDaoPreactivatedAllSuite) BlockchainOpts() []config.BlockchainOption {
+	return []config.BlockchainOption{
+		config.WithFeatureSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_1miner_dao.json",
+		),
+		config.WithRewardSettingFromFile(
+			rewardSettingsFolder,
+			boostRewardSettingsFolder,
+			"boost_reward_preactivated_14_19_20_21_22_23_1miner_dao.json"),
```

Comment:
Suggest to add new option which can combine `WithFeatureSettingFromFile` and `WithRewardSettingFromFile`